nexus/reconfigurator/planning/src/blueprint_builder.rs

```diff
@@ -138,6 +143,48 @@ pub struct BlueprintBuilder<'a> {
 }
 
 impl<'a> BlueprintBuilder<'a> {
+    /// Build a Blueprint describing a system with no Omicron zones
+    pub fn build_initial_empty(policy: &Policy, creator: &str) -> Blueprint {
+        Self::build_initial_empty_impl(
+            policy,
+            creator,
+            BlueprintBuilderRng::new(),
+        )
+    }
```

Comment:
Can this return a `BlueprintBuilder` which `build()` can then be called on? Not sure how useful that might be.

Reply:
It could!  But we don't have any consumers for that yet so I'm not sure that'll make sense.  It will also add an extra step to all the callers but that's okay.